internal/helm/helm.go

```diff
@@ -408,30 +412,41 @@ const (
 	StatusNotReady ReleaseStatus = "not-ready"
 )
 
-// Status will check for the readyness of the release returning an internal status instead of
-// the Helm release status (https://github.com/helm/helm/blob/main/pkg/release/status.go).
-// Helm is not checking for the actual status of the release and even if the resources are still
-// in deployment they will be marked as "deployed"
-func Status(ctx context.Context, logger logr.Logger, cluster *kubernetes.Cluster, namespace, releaseName string) (ReleaseStatus, error) {
+// Release returns the named Helm release, including the associated resources. This is the internal
+// equivalent of the `helm status` command.
+func Release(ctx context.Context, logger logr.Logger, cluster *kubernetes.Cluster,
+	namespace, releaseName string) (*helmrelease.Release, error) {
+
 	helmClient, err := GetHelmClient(cluster.RestConfig, logger, namespace)
 	if err != nil {
-		return StatusUnknown, err
+		return nil, err
 	}
 
-	releaseStatus, err := helmClient.Status(releaseName)
+	release, err := helmClient.Status(releaseName)
 	if err != nil {
-		return StatusUnknown, errors.Wrapf(err, "getting release status %s - %s", namespace, releaseName)
+		return nil, errors.Wrapf(err, "getting release status %s - %s", namespace, releaseName)
 	}
 
-	resourceList := getResourceListFromRelease(releaseStatus)
+	return release, err
+}
+
+// Status will check for the readyness of the release returning an internal status instead of
```

Comment:
Heh. I had made that change actually and then reverted to what you see, the original text ... Checked with some online dict that both forms are ok.

Reply:
Not sure where did you check because every dictionary, or Google, are saying that `readyness` is incorrect:
- https://dictionary.cambridge.org/spellcheck/english/?q=readyness -> https://dictionary.cambridge.org/dictionary/english/readiness
- https://www.dictionary.com/browse/readyness -> https://www.dictionary.com/browse/readiness
- https://www.thesaurus.com/browse/readiness